katportalclient/client.py

```diff
@@ -1130,7 +1125,7 @@ def sensor_names(self, filters):
         SensorNotFoundError:
             - If any of the filters were invalid regular expression patterns.
         """
-        url = self.sitemap['historic_sensor_values'] + '/sensors'
+        url = self.sitemap['historic_sensor_values'] + '/api/sensors'
```

Comment:
An alternative is to change the sitemap on katportal to include the `/api` part.  Not sure if it makes much difference.